test/ex_doc/formatter/html/erlang_test.exs

```diff
@@ -54,7 +53,11 @@ defmodule ExDoc.Formatter.HTML.ErlangTest do
       extras: []
     ]
 
-    ExDoc.generate_docs(config[:project], config[:version], config)
+    ## Silence warnings generated by docs
```

Comment:
I'm not sure how to do that... I've pushed a version that removed the silence but I think I need a bit of help figuring out how to selectively silence only what we want

Reply:
@garazdawi I don't we need to worry about it too much. Something I wanted to do for a long time was for these modules to _return_ warnings instead of printing them right away so the problem will go away. I'm happy to work on that after this lands.